identifiers/country-us/state-wa-local_gov.csv

```diff
@@ -152,3 +152,6 @@ ocd-division/country:us/state:wa/place:kennewick/ward:3,Kennewick WA Ward 3
 ocd-division/country:us/state:wa/place:raymond/ward:1,Raymond WA Ward 1
 ocd-division/country:us/state:wa/place:raymond/ward:2,Raymond WA Ward 2
 ocd-division/country:us/state:wa/place:raymond/ward:3,Raymond WA Ward 3
+ocd-division/country:us/state:wa/county:clark/council_district:4,Clark County Council District 4
```

Comment:
Looks like the other council districts are missing for Clark County, WA

Reply:
Districts 1-3 should be there, I updated district 4's name to match the precedent for districts 1-3 though